tests/js/sentry-test/enzyme.js

```diff
@@ -4,16 +4,16 @@ import {cache} from 'emotion'; // eslint-disable-line emotion/no-vanilla
 import {ThemeProvider} from 'emotion-theming';
 import {mount, render, shallow} from 'enzyme'; // eslint-disable-line no-restricted-imports
 
-import theme from 'app/utils/theme';
+import {lightTheme} from 'app/utils/theme';
 
 const mountWithTheme = (tree, opts) => {
   const WrappingThemeProvider = props => (
     <CacheProvider value={cache}>
-      <ThemeProvider theme={theme}>{props.children}</ThemeProvider>
+      <ThemeProvider theme={lightTheme}>{props.children}</ThemeProvider>
     </CacheProvider>
   );
 
   return mount(tree, {wrappingComponent: WrappingThemeProvider, ...opts});
 };
 
-export {mount, mountWithTheme, render, shallow};
+export {mountWithTheme as mount, mountWithTheme, render, shallow};
```

Comment:
are you planning to replace all `mount` with `mountWithTheme` in a follow-up PR?

Reply:
I think that's an option, but I wasn't specifically planning on it. @markstory do you have an opinion?

Reply:
I think our lives will be easier long term if our test helpers always attach the theme. I know I've lost at least a few hours to updating tests that have failed because a component deep in the render tree now needs theme.

Reply:
right, i agree with that and i think everyone here does. my question though was "should we rename all the instances of mount to mountWithTheme, or just use this alias?"

Reply:
I believe that we always try to avoid renaming imports, so I think that renaming all instances of mount would be a good idea.